docker/Dockerfile.frontend

```diff
@@ -1,11 +1,15 @@
-FROM node:16.13.0 AS frontend
+FROM node:16.13.0-slim AS frontend
 
 WORKDIR /app/oppia
 
-RUN apt-get update -y && apt-get upgrade -y \
+RUN apt-get update -y && apt-get install -y --no-install-recommends \
     chromium \
     curl \
-    git
+    git \
+    ca-certificates && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+RUN git config --global http.sslVerify "true"
```

Comment:
why do we need to set this manually? it defaults to true per the docs: https://git-scm.com/docs/git-config/2.22.0

Reply:
Yes , --global is set by default so we donot need to set up manually.